tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -47,6 +47,8 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
             CanBeToggled = !enabledEnvVar.ConfigurationResult.IsValid;
             AppsecEnabled = enabledEnvVar.WithDefault(false);
 
+            ApmTracingEnabled = config.WithKeys(ConfigurationKeys.ApmTracingEnabled).AsBool(true);
```

Comment:
As discussed, I think duplicating this here for now makes sense, and we can look at extracting it more cleanly later